tests/common/tests/kalman_fitting_test.cpp

```diff
@@ -85,7 +85,7 @@ void KalmanFittingTests::ndf_tests(
     const fitting_result<traccc::default_algebra>& fit_res,
     const track_state_collection_types::host& track_states_per_track) {
 
-    scalar dim_sum = 0;
+    unsigned int dim_sum = 0;
```

Comment:
Ah btw, I meant this to be scalar object rather than scalar. In some extensions of fitter, it is the summation of (dimension * weight) where the weight is scalar

Reply:
Ah okay, that makes sense. I'll change it back